llvm/lib/Target/DirectX/DXILOpBuilder.cpp

```diff
@@ -131,8 +151,9 @@ struct OpCodeProperty {
   dxil::OpCodeClass OpCodeClass;
   // Offset in DXILOpCodeClassNameTable.
   unsigned OpCodeClassNameOffset;
-  uint16_t OverloadTys;
-  llvm::Attribute::AttrKind FuncAttr;
+  std::vector<OpOverload> Overloads;
+  std::vector<OpStage> Stages;
+  std::vector<OpAttribute> Attributes;
```

Comment:
We should probably use `llvm::SmallVector` for these to avoid unnecessary allocations.

Reply:
> We should probably use `llvm::SmallVector` for these to avoid unnecessary allocations.

Changed.